src/components/Authorship.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div id="author-container">
+    <p>
+      <span id="primary-author-statment">
+        The development was lead by 
```

Comment:
Include the title in this first sentence..."The development of "From Snow to Flow" was lead by..."